docs/whitepaper.tex

```diff
@@ -220,6 +220,16 @@ \subsection{DEX pool}
   \item A boolean representing the pool's kill switch
 \end{itemize}
 
+\subsection{Initializing the protocol}
+
+The first step to initialize the protocol, is to mint an NFT, which will be
+considered the `master' identifier of the protocol, namely the NFT identifying
+the Config UTXO. Next, using the name of this NFT, we can calculate the hash
+associated with the three other scripts (pool validator script, liquidity token
+minting policy and Pool ID minting policy). Finally, a Config UTXO is
+initialized with the `master' NFT and a datum that contains the three hashes, in
+that order: Pool script, liquidity token script and pool ID script.
```

Comment:
IIRC in the meeting we agreed it was fine to emit the pool id token minting policy from the config utxo datum, can we remove it here in the spec?